File: 코딩 기초 트레이닝/x 사이의 개수-p181867/x 사이의 개수.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181867

def solution(myString):
    answer = []
    lens = 0

    for s in myString:
        if s != 'x':
            lens += 1
        else:
            answer.append(lens)
            lens = 0

    answer.append(lens)
    return answer

File: 코딩 기초 트레이닝/x 사이의 개수-p181867/x 사이의 개수.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181867

function solution(myString) {
    const answer = [];
    let lens = 0;

    myString.split("").forEach((m) => {
        if(m !== 'x') {
            lens++;
        }
        else {
            answer.push(lens);
            lens = 0;
        }
    });

    answer.push(lens);
    return answer;
}
